dali/kernels/imgproc/color_manipulation/equalize/lut_test.cc

```diff
@@ -163,8 +163,11 @@ TEST_F(EqualizeLutGpuTest, SinglePoint) {
   auto in_view = in_.cpu();
   for (int sample_idx = 0; sample_idx < batch_shape.num_samples(); sample_idx++) {
     for (int channel_idx = 0; channel_idx < batch_shape[sample_idx][0]; channel_idx++) {
-      in_view[sample_idx].data[channel_idx * range_size + sample_idx * 71 + channel_idx] =
-          123 * channel_idx + sample_idx;
+      ASSERT_EQ(batch_shape[sample_idx][1], range_size);
+      for (int i = 0; i < range_size; i++) {
+        in_view[sample_idx].data[channel_idx * range_size + i] =
+            (i == 71 + channel_idx) ? ((sample_idx << 11) + channel_idx) : 0;
```

Comment:
This code looks almost as if intentionally obfuscated.
```suggestion
        int index_of_nonzero_value = 71 + channel_idx;
        int nonzero_value = (sample_idx << 11) + channel_idx;
        in_view[sample_idx].data[channel_idx * range_size + i] =
            i == index_of_nonzero_value ? nonzero_value : 0;
```

Reply:
fixed